.github/workflows/scripts/supported-versions.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+# Prints supported versions based on the current release branch targeted
+# Version output is in JSON
+
+set -e
+set -x
+
+if git merge-base --is-ancestor origin/release/v5.0 HEAD
+then
+  echo -n "[\"v1.23.9-k3s1\", \"v1.29.4-k3s1\"]"
```

Comment:
This is slightly different than the 2.9 version - since this script seems to cover all releases, should we make sure the script is the same on each branch?